Aftonbladet.js

```diff
@@ -14,7 +14,7 @@
 
 /*
 	Aftonbladet Translator - Parses Aftonbladet articles and creates Zotero-based metadata.
-	Copyright (C) 2016 Jonatan Svensson Glad
+	Copyright (C) 2016 Jonatan Svensson Glad & Philipp Zumstein
```

Comment:
> for all help, and code blocks I've stolen

Well, when I am sharing code blocks here, then I want, that you use them. Information must be shared to be useful, and I am happy if you can reuse some of these. Credit is fine and sometimes can be useful...